### PR TITLE
Fix monster waves

### PR DESCRIPTION
**Pull request by gerotib**
Fixes monsters that should use their specific lua spells - mainly wrong spell areas.

### CODE REVIEW

File: data/spells/scripts/monster/arachnophobica waveenergy.lua [hunk 1]
```diff
@@ -1,5 +1,5 @@
 local combat = createCombatObject()
-combat:setParameter(COMBAT_PARAM_TYPE, COMBAT_ENERGY)
+combat:setParameter(COMBAT_PARAM_TYPE, COMBAT_ENERGYDAMAGE)
```
Comment (dex-89) [inline data/spells/scripts/monster/arachnophobica waveenergy.lua:2]: #2365 duplication

File: data/monster/magicals/guzzlemaw.lua [hunk 2]
```diff
@@ -113,7 +113,7 @@ monster.attacks = {
 	{name ="melee", interval = 2000, chance = 100, minDamage = 0, maxDamage = -499},
 	-- bleed
 	{name ="condition", type = CONDITION_BLEEDING, interval = 2000, chance = 10, minDamage = -500, maxDamage = -1000, radius = 3, target = false},
-	{name ="combat", interval = 2000, chance = 10, type = COMBAT_PHYSICALDAMAGE, minDamage = 0, maxDamage = -900, length = 8, spread = 3, effect = CONST_ME_EXPLOSIONAREA, target = false},
+	{name ="combat", interval = 2000, chance = 10, type = COMBAT_PHYSICALDAMAGE, minDamage = 0, maxDamage = -900, length = 8, spread = 3, effect = CONST_ME_EXPLOSIONAREA, target = true},
```
Comment (hyresu) [inline data/monster/magicals/guzzlemaw.lua]: Shouldn't this spell be a wave, which is not targeted?

File: data/spells/scripts/monster/arachnophobica waveenergy.lua (hunk 1 above)
Comment (dex-89) [inline data/spells/scripts/monster/arachnophobica waveenergy.lua]: #2365 duplication

**Review by dex-89 — CHANGES_REQUESTED** [review]

File: data/monster/bosses/kraknaknork.lua [hunk 3]
```diff
@@ -77,9 +77,9 @@ monster.loot = {
 
 monster.attacks = {
 	{name ="melee", interval = 2000, chance = 100, minDamage = 0, maxDamage = -10, condition = {type = CONDITION_POISON, totalDamage = 5, interval = 4000}},
-	{name ="combat", interval = 2000, chance = 10, type = COMBAT_ICEDAMAGE, minDamage = -1, maxDamage = -15, length = 7, spread = 3, effect = CONST_ME_ICEAREA, target = false},
-	{name ="combat", interval = 2000, chance = 10, type = COMBAT_EARTHDAMAGE, minDamage = -1, maxDamage = -10, length = 6, spread = 3, effect = CONST_ME_SMALLPLANTS, target = false},
-	{name ="combat", interval = 2000, chance = 10, type = COMBAT_PHYSICALDAMAGE, minDamage = -1, maxDamage = -12, length = 7, spread = 3, effect = CONST_ME_EXPLOSIONAREA, target = false},
+	{name ="kraknaknork ice wave", interval = 2000, chance = 10, minDamage = -1, maxDamage = -15, target = false},
+	{name ="kraknaknork poison wave", interval = 2000, chance = 10, minDamage = -1, maxDamage = -10, target = false},
+	{name ="kraknaknork explosion wave", interval = 2000, chance = 10, minDamage = -1, maxDamage = -12, target = false},
```
Comment (dex-89) [inline data/monster/bosses/kraknaknork.lua]: @ElTeixe: you are working on rookgaard. Can you validate?

Comment (gerotib) [reply data/monster/bosses/kraknaknork.lua]: I assume that thumb up confirmed it's valid

Comment (dex-89) [reply data/monster/bosses/kraknaknork.lua]: No, he let me know that he had read and wrote to me that if he got to this boss, he would check it.

Comment (ElTeixe) [reply data/monster/bosses/kraknaknork.lua]: The changes are correct except "kraknaknork explosion wave" minDamage should be 0. But I can correct it later, I have to edit other Kraknaknork things.

**Review by ElTeixe — APPROVED** [review]

**Review by majestyotbr — APPROVED** [review]

**Review by dex-89 — APPROVED** [review]

**Pull request merged**